README.md

```diff
@@ -55,7 +55,7 @@ var dir = requireDir('./path/to/dir', {recurse: true});
 ## Options
 
 `recurse`: Whether to recursively `require()` subdirectories too.
-Default is false.
+Default is false (`node_modules` in the subdirectories will be ignored).
```

Comment:
Minor: let's clarify that `node_modules` will be ignored if this is true, not false.

```md
`recurse`: Whether to recursively `require()` subdirectories too.
(`node_modules` within subdirectories will be ignored.)
Default is false.
```